app/templates/thank-you.html

```diff
@@ -2,6 +2,8 @@
 
 {% block page_title %}Submission Successful{% endblock %}
 
+{% block subheader %}{% endblock subheader %}
```

Comment:
This will remove the 'previous' link too as it's in that block.

There's a value `navigation` that is exposed to `layouts/_base.html` that is supposed to switch the nav on/off. Can this be used instead?